README.md

```diff
@@ -95,7 +95,7 @@ Add to `config/config.exs` file:
 config :kanta,
```

Comment:
https://github.com/curiosum-dev/kanta-example-app/blob/develop/config/config.exs#L66
Seems like it should be 
```elixir
config :my_app, Kanta
```

Reply:
Also add a note about deleting `_build`, got an error that made no sense myself due to compile cache